src/Discovery/Discovery.tsx

```diff
@@ -745,7 +746,10 @@ const Discovery: React.FunctionComponent<Props> = (props: Props) => {
                   config={props.config}
                   studies={props.studies}
                   filterState={filterState}
-                  setFilterState={setFilterState}
+                  setFilterState={(event) => {
+                    props.onAdvancedSearch(event);
```

Comment:
better to only trigger this if the pagniation page is not 1?

Reply:
i dont think the extra code is needed (as the state would not change if already on page 1), would like to see setFilterState using the existing reducers but dont think a refactor is worth our time